src/features/import-assertions.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Import assertions'
+author: 'Dan Clark ([@dandclark1](https://twitter.com/dandclark1))'
```

Comment:
Ah, and one final suggestion: do you want to add a funny (on-topic or not) tagline here? You can see other blog posts for examples.